mod/config/pkg/config/config.go

```diff
@@ -113,17 +113,3 @@ func GetConfig(v *viper.Viper) (Config, error) {
 	}
 	return *conf, nil
 }
-
-// ValidateBasic returns an error if min-gas-prices field is empty in
-// BaseConfig. Otherwise, it returns nil.
-func (c Config) ValidateBasic() error {
-	// if c.Pruning == pruningtypes.PruningOptionEverything &&
-	// c.StateSync.SnapshotInterval > 0 {
-	// 	return sdkerrors.ErrAppConfig.Wrapf(
-	// 		"cannot enable state sync snapshots with '%s' pruning setting",
-	// pruningtypes.PruningOptionEverything,
-	// 	)
-	// }
```

Comment:
These may be checks we want to keep? Should take this opportunity to determine if any validation is necessary for Comet config